etc/helm/examples/local-dev-values.yaml

```diff
@@ -38,3 +38,9 @@ etcd:
 postgresql:
   service:
     type: NodePort
+
+jaeger:
+  enabled: true
```

Comment:
I'll update this to false

Reply:
This is perhaps a product of my unfamiliarity with Helm, but how would one deploy only Jaeger into an existing cluster? The way at least I've used this in the past is:
- User is having some kind of performance problem
- We walk them through deploying Jaeger into their cluster
- We walk them through collecting a trace

I'm basically hoping to check that this workflow still works/how it works?

Reply:
I believe the right way to handle this is to use `helm upgrade` and pass the updated `override-values.yaml`. I'll test to see that this works

Reply:
In that case, I'd prefer to do it as a separate install